### PR TITLE
feat: added prettier codestyle for ts scripts

### PR DESCRIPTION
**Pull request by hritikgupta**

### CODE REVIEW

File: tests/assemblyscript/package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "asc": "asc",
     "lint": "eslint . --ext .ts",
-    "build": "ts-node buildscripts/build.ts"
+    "build": "ts-node buildscripts/build.ts",
+    "prettier-format-check": "prettier '**/*.ts' --check",
```
Comment (loganek) [inline tests/assemblyscript/package.json:11]: should we also update CI build scripts to enforce the format?

Comment (hritikgupta) [reply tests/assemblyscript/package.json]: fixed!

Comment (hritikgupta) [reply tests/assemblyscript/package.json]: added!

**Review by loganek — APPROVED** [review]
LGTM

**Pull request merged**